nomad/resource_job_test.go

```diff
@@ -1736,16 +1735,17 @@ func testResourceJob_csiControllerCheck(s *terraform.State) error {
 		return fmt.Errorf("task group %s not found", taskGroupName)
 	}
 
-	expCSIPluginConfig := api.TaskCSIPluginConfig{}
-	json.Unmarshal([]byte(`{
-        "ID": "aws-ebs0",
-        "Type": "controller",
-        "MountDir": "/csi"
-	}`), &expCSIPluginConfig)
```

Comment:
I'm not sure why we used to do this JSON decode thing, I found it a little harder to update tests since you can't rely on the actual struct information for things like autocomplete and code navigation.